src/org/javarosa/core/api/ILogger.java

```diff
@@ -15,38 +15,69 @@
  */
 
 /**
- * 
+ *
  */
 package org.javarosa.core.api;
 
-import org.javarosa.core.log.IFullLogSerializer;
-import org.javarosa.core.log.StreamLogSerializer;
-
 import java.io.IOException;
 import java.util.Date;
+import org.javarosa.core.log.IFullLogSerializer;
+import org.javarosa.core.log.StreamLogSerializer;
 
 /**
  * IIncidentLogger's are used for instrumenting applications to identify usage
  * patterns, usability errors, and general trajectories through applications.
- * 
- * @author Clayton Sims
- * @date Apr 10, 2009 
  *
+ * @author Clayton Sims
+ * @deprecated Use {@link org.slf4j.LoggerFactory#getLogger(Class)} instead
  */
+@Deprecated
```

Comment:
If you deprecate the type, must you also deprecate all the members?

Reply:
It's just a pretext to add the `@deprecated` JavaDoc so that anyone ctrl-clicking gets information about an alternative...